README.md

```diff
@@ -17,7 +17,7 @@ HOF provides the following which can be accessed through its properties.
  * [hmpo-template-mixins](https://github.com/UKHomeOffice/passports-template-mixins)
  * [hof-controllers](https://github.com/UKHomeOffice/hof-controllers)
  * [i18n-future](https://github.com/lennym/i18n-future)
- * [middleware](https://github.com/UKHomeOffice/hof/blob/master/documentation/middleware.md)
+ * [middleware](./documentation/middleware.md)
```

Comment:
Are you sure this will still work on the GitHub project page?
Is GitHub clever enough to fix this? (I suppose we can merge it and find out!)